CollectionManager.js

```diff
@@ -71,22 +71,28 @@ class CollectionManager {
   }
 
   _addCloseListener() {
-    this.changeStream.on('close', async () => {
+    this.changeStream.on('close', () => {
       console.log('close event');
     });
   }
 
   _addErrorListener() {
     this.changeStream.on('error', (error) => {
-      console.log('changeStream error', error);
+      console.log(this.collection,' changeStream error', error);
+      // resume of change stream was not possible, as the resume token was not found
+      if (error.code === 40585 || error.code === 40615) {
+        this.resetChangeStream();
+      }
     });
   }
 
-  async resetChangeStream() {
+  async resetChangeStream(dump=false) {
```

Comment:
a linter would tell you to do `(dump = false)`. Can you also fix other instances of this?